포스트 유저 정보 구조 잡기

beomseok37 opened the pull request:
### 제목
포스트 유저 정보 구조 잡기

### 파일 변경
- frontend/src/components/Post/index.tsx : 유저 정보 칸 구조 잡음
-  frontend/src/components/buttons/ProfileImageButton/index.tsx : props 설정
-  frontend/src/components/buttons/ProfileImageButton/style.ts : style 적용
-  frontend/src/components/buttons/profileUserNameButton/index.tsx : props 설정
-  frontend/src/components/buttons/profileUserNameButton/style.ts : style 적용

beomseok37 committed:
Feat: profileUserNameButton 생성

- username클릭 시 해당 user의 프로필 페이지로 이동

beomseok37 committed:
Feat: ProfileImageButton 구조 잡기

- props로 img src, href 받기

beomseok37 committed:
Feat: post profile정보 구조 잡기

- Row로 감싸기
- ProfileImageButton, ProfileUserNameButton으로 버튼 나누기

hyunmindev commented on frontend/src/components/Post/index.tsx:
```diff
@@ -15,7 +16,10 @@ import { Wrapper, Buttons } from './style';
     <Wrapper>
-      <ProfileImageButton href='users/123'>image, username</ProfileImageButton>
+      <Row justifyContent='flex-start'>
+        <ProfileImageButton href='users/123' imgSrc='/images/logo.svg' usage='post' />
+        <ProfileUserNameButton href='user/123' userName='tiger' usage='post' />
```
이 두개를 `ProfileButton` 컴포넌트로 합치면 `useage` props가 필요없어질 것 같습니다

beomseok37 replied:
아 그렇네요...........

beomseok37 committed:
Refactor: 리뷰 반영

- usage props 삭제
- ProfileImageButton, ProfileUserNameButton을 합친 ProfileButton
  component 생성

hyunmindev commented on frontend/src/components/buttons/post/ProfileButton/index.tsx:
```diff
@@ -0,0 +1,18 @@
+function profileButton() {
+  return (
+    <Wrapper>
+      <Row justifyContent='flex-start'>
+        <ProfileImageButton href='users/123' imgSrc='/images/logo.svg' />
+        <ProfileUserNameButton href='user/123' userName='tiger' />
+      </Row>
```
프로필 버튼으로 합치면 이미지랑 이름 표시만 하면 될것같아요~~

beomseok37 committed:
Refactor: 리뷰 반영

- Profile component 이름 변경
